src/content/blog/general/aws-irsa/index.md

```diff
@@ -4,6 +4,7 @@ title: Verifying images in a private Amazon ECR with Kyverno and IAM Roles for S
 linkTitle: Verifying images in a private Amazon ECR with Kyverno and IAM Roles for Service Accounts (IRSA)
 author: Shuting Zhao
 description: Using Kyverno to verify images with IRSA
+category: General
```

Comment:
Why do we have a new "general" category and why are we adding it to all these blog posts?